azure-storage-queue/src/test/scala/akka/stream/alpakka/azure/storagequeue/scaladsl/AzureQueueSpec.scala

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.azure.storagequeue.scaladsl
+
+import org.scalatest._
+import com.microsoft.azure.storage._
+import com.microsoft.azure.storage.queue._
+import scala.concurrent._
+import scala.concurrent.duration._
+import akka.stream.scaladsl._
+import akka.stream._
+import akka.testkit._
+import akka.actor.ActorSystem
+import scala.util.Properties
+import akka.stream.alpakka.azure.storagequeue._
+import scala.collection.JavaConverters._
+
+
+// These tests are all live since the Azure Storage Emulator
+// does not run on Linux/Docker yet
```

Comment:
Does this mean the tests will not be able to run on travis or that they need an actual azure account for this to work?